content/blogposts/blog-post-starter-pack.md

```diff
@@ -0,0 +1,255 @@
+
+<Figure 
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/default_hero_social.png"
+  alt="Sourcegraph thumbnail"
+  caption="The Sourcegraph logo"
+/>
+
+
+### TableWrapper
+
+Tables are a straightforward way to present and compare data. When preparing a table, please include the `TableWrapper` component to ensure that the table adapts to different screen sizes. To support your preference, our blog accepts tables written in both Markdown and HTML.
+
+This table is written in **Markdown**:
+
+<TableWrapper>
+| Id   | Species     | Sepal Length (cm) | Sepal Width (cm) | Petal Length (cm) | Petal Width (cm) |
+|------|-------------|-------------------|------------------|-------------------|------------------|
+| 1    | Iris Setosa | 5.1               | 3.5              | 1.4               | 0.2              |
+| 2    | Iris Setosa | 4.9               | 3.0              | 1.4               | 0.2              |
+</TableWrapper>
+
+This is the same table, but written in **HTML**:
+
+<TableWrapper>
+  <table>
+    <thead>
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Sepal Length (cm)</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1</td>
+        <td>Iris Setosa</td>
+        <td>5.1</td>
+        <td>3.5</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+      <tr>
+        <td>2</td>
+        <td>Iris Setosa</td>
+        <td>4.9</td>
+        <td>3.0</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+    </tbody>
+  </table>
+</TableWrapper>
+
+
+### YouTube
+
+You can easily embed YouTube videos using the `YouTube` component. The `id` value is found at the end of the URL for any YouTube video. To display the title below the video, add the `showTitle` property.
+
+<YouTube
+  title="Accelerate developer onboarding with Sourcegraph" 
+  id="DgwvhRW1Cbc"
+  showTitle={true}
+/>
+
+
+### Video
+
+The `Video` component supports .mp4 and .webm files. First, upload the two versions of the video to Google Cloud. Then, add the root names of these files to the `source` property, without including their extensions. You can choose to repeat a video by including the `loop` property, or display the caption using the `showCaption` property.
+
+<Video 
+  source={{webm: 'blog/shift-left/01-code-reuse', mp4: 'blog/shift-left/01-code-reuse'}} 
+  loop={true}
+  title="Code reuse"
+  caption="An example of code search with Sourcegraph"
+  showCaption={true}
+/>
+
+
+### HubSpot Form Example
+
+Use the `EmbeddedHubSpot` component to drop a custom HubSpot form into your post. In most cases, the `formId` is the only property that needs to be updated. This is the same `formId` dictated by the HubSpot console. Once added to your post, the HubSpot form will render wherever you place the `div` using the specified `id` value.
+
+<EmbeddedHubSpot 
+  portalId='2762526' 
+  formId='08e6c442-0e7c-4892-a262-76dae55ab497' 
+  targetId='#sign-up-form' />
+
+<div id="sign-up-form"></div>
```

Comment:
This API has changed and the component is now called `HubSpotForm`

```suggestion
### HubSpot Form Example

Use the `HubSpotForm` component to drop a custom HubSpot form into your post. In most cases, the `masterFormName` and `chiliPiper` props are the only properties that need to be updated. Once added to your post, the HubSpot form will render in its place. All the options for this component are:

- `formId` - an optional form ID if the form doesn't fall under one of our [master forms](https://docs.google.com/document/d/1qa393uAW_tjwNtH9jHf_d__PCJgmbsxNoxi9cO4XOcE/edit)
- `masterFormName` - an optional master form name; `contactMulti`, `contactEmail`, `gatedMulti`, or `gatedEmail` for general contact forms and gated forms that have multi fields or an email field
- `onFormSubmitted` - an optional callback function for triggering an action after a form is submitted
- `inlineMessage` - an optional message to display after a form is submitted. We have a [generic fallback](https://github.com/sourcegraph/about/blob/main/src/components/HubSpotForm.tsx#L288) if none is provided. 
- `chiliPiper` - an optional boolean (`true`/`false`) to enable Chili Piper. This is false by default.

<HubSpotForm
  masterFormName="contactEmail"
  chiliPiper={true}
/>
```

Reply:
Thank you!